components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -18,6 +18,7 @@
 
 package org.wso2.carbon.identity.scim2.common.impl;
 
+import edu.emory.mathcs.backport.java.util.Collections;
```

Comment:
shall we use the apache commons Collection instead of this?

Reply:
Fixed: f999948935ff2081b9362c4cbe0d6e8f5ee238db